vtexdocs/RapiDoc
# Chore/update rapidoc mini

#### What is the purpose of this pull request?

To update the rapidoc-mini so it contains all the expected elements (that were already included into the default rapidoc tag, which we replaced by the mini version).

#### What problem is this solving?

The absence of some elements at the API references and related styling problems since we switched from rapidoc to rapidoc-mini.

#### How should this be manually tested?

[Access this deploy preview](https://deploy-preview-78--elated-hoover-5c29bf.netlify.app/) and check if:
- The method-path div (with copy button upon the path) appears below the API references title
- Rapidoc-mini's default colored borders around the API references are gone
- The font-size and spaces between the API reference description, horizontal separators and its cards are balanced as expected (usually 24px)

[Further information about this contribution here.](https://www.notion.so/vtexhandbook/Atualizar-conte-do-Lit-retornado-pelo-rapidoc-mini-para-conter-todos-os-elementos-previstos-no-figma-4ab391115a104bb98a8f8bd05297f698)

#### Screenshots or example usage

| Before | After |
| ----------- | ----------- |
| ![image](https://user-images.githubusercontent.com/39717968/193375844-7875b316-c4f6-4276-9feb-51ebee5125b6.png)  | ![image](https://user-images.githubusercontent.com/39717968/193375871-01d07844-3aba-42e1-b3dd-0a908605d70d.png) |
| ![image](https://user-images.githubusercontent.com/39717968/193376002-ef3f52b8-c5d6-441c-b792-c04f77779643.png)  | ![image](https://user-images.githubusercontent.com/39717968/193376022-efea5e4c-0ab8-4ed8-a57f-727fe39568e5.png) |
